### PR TITLE
Fix Data Connect template

### PR DESCRIPTION
**Pull request by rodydavis**

### CODE REVIEW

File: dataconnect/dev.nix [hunk 1]
```diff
@@ -31,12 +30,13 @@
...
       workspace = {
         onCreate = {
+          update-firebase = "npm install -g firebase-tools";
           postgres = ''
             psql --dbname=postgres -c "ALTER USER \"user\" PASSWORD 'mypassword';"
             psql --dbname=postgres -c "CREATE DATABASE dataconnect;"
             psql --dbname=dataconnect -c "CREATE EXTENSION vector;"
           '';
-          npm-install = "pnpm i --prefix=./webapp";
+          npm-install = "cd webapp && npm i && npm i firebase@dataconnect-preview";
```
Comment (maneesht) [inline dataconnect/dev.nix:39]: `dataconnect-preview` no longer needed

File: dataconnect/nextjs-blank/webapp/package.json [hunk 2]
```diff
@@ -9,12 +9,13 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@blank/generated": "file:dataconnect-generated",
+		"firebase": "^10.12.4-dataconnect-preview.d986d4bf2",
```
Comment (rodydavis) [inline dataconnect/nextjs-blank/webapp/package.json:13]: @maneesht what will this need to be?

File: dataconnect/nextjs-email-app/webapp/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
 	},
 	"dependencies": {
 		"@email-app/email": "file:email-generated",
-		"firebase": "10.12.0-dataconnect-preview.388b61c7e",
+		"firebase": "^10.12.4-dataconnect-preview.d986d4bf2",
```
Comment (rodydavis) [inline dataconnect/nextjs-email-app/webapp/package.json:13]: @maneesht what will this need to be?

File: dataconnect/nextjs-blank/webapp/package.json (hunk 2 above)
Comment (maneesht) [reply dataconnect/nextjs-blank/webapp/package.json:13]: ^10.14.0

File: dataconnect/nextjs-email-app/webapp/package.json (hunk 3 above)
Comment (maneesht) [reply dataconnect/nextjs-email-app/webapp/package.json:13]: ^10.14.0

**Review by maneesht — APPROVED** [review]

**Pull request merged**